tests/addremove.test.js

```diff
@@ -0,0 +1,27 @@
+const { addNewItem, removeItem } = require('../src/__mocks__/addremove');
+
+const taskList = [
+  {
+    description: 'Task 1',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Task 2',
+    completed: false,
+    index: 1,
+  },
+];
+
+describe('Tests for add remove', () => {
+  test('Add Item', () => {
+    addNewItem('New Task', false, taskList.length, taskList);
+    expect(taskList.length).toBe(3);
+  });
+  test('Remove Item', () => {
+    expect(removeItem(1, taskList)).toBe(2);
+  });
+  test('Check The Index', () => {
+    expect(taskList[1].index).toBe(1);
+  });
+});
```

Comment:
- [ ] Good job so far implementing these unit tests using mock files. But I recommend you to look carefully part of your project requirement below. As the actual `addNewItem` and `removeItem` functions manipulate local storage, I recommend you import these actual functions to your test module and mock the `localStorage`. :pray: 
``` 
Some of the functions however will update localStorage and manipulate the DOM. For those, you will need to use mocks:
- Mock a storage object to "imitate" localStorage operations.
- Mock HTML to test if add/delete functions add or remove exactly one <li> element to/from the list in the DOM.
```

Reply:
Thanks, Sir For the feedback! We have implemented it!